README.md

```diff
@@ -20,7 +20,7 @@ If you've never used Github here's a [Github101.md](docs/GITHUB101.md) to get yo
 
 Everything is co-ordinated through github issues.  The current development process is:
 
-👨‍⚕️👩‍⚕️ **Greg Miller** and other Western Health staff provide features, content and suggestions via GitHub issues.  Greg & Co can create new issues for features to add, things that need changing, etc.  
+👨‍⚕️👩‍⚕️ **Gregg Miller** and other Western Health staff provide features, content and suggestions via GitHub issues.  Greg & Co can create new issues for features to add, things that need changing, etc.  
```

Comment:
Doh!  I don't belive there was a typo in your name!

Reply:
```suggestion
👨‍⚕️👩‍⚕️ **Gregg Miller** and other Western Health staff provide features, content and suggestions via GitHub issues.  Gregg & Co can create new issues for features to add, things that need changing, etc.  
```